mdk/build.gradle

```diff
@@ -65,16 +69,8 @@ minecraft {
         server {
             workingDirectory project.file('run')
 
-            // Recommended logging data for a userdev environment
-            // The markers can be added/remove as needed separated by commas.
-            // "SCAN": For mods scan.
-            // "REGISTRIES": For firing of registry events.
-            // "REGISTRYDUMP": For getting the contents of all registries.
             property 'forge.logging.markers', 'REGISTRIES'
 
-            // Recommended logging level for the console
-            // You can set various levels here.
-            // Please read: https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels
```

Comment:
While I'm not going to suggest you do this here, I did have the idea (and in fact, [implemented it](https://github.com/SocketMods/mod-template/blob/78e12eb09a48b0deaefa7df19dbf10123093eba9/build.gradle#L27-L37) in SocketMod's mod template) to use [`DomainObjectCollection#configureEach​`](https://docs.gradle.org/current/javadoc/org/gradle/api/DomainObjectCollection.html#configureEach-org.gradle.api.Action-) to configure all of these arguments for all run configurations in one place, rather than fragmented across each run config. Just food for thought.